test/fixtures/dialects/spark3/insert_table.sql

```diff
@@ -0,0 +1,66 @@
+-- Single Row Insert Using a VALUES Clause
+INSERT INTO TABLE students VALUES
+('Amy Smith', '123 Park Ave, San Jose', 111111);
+
+INSERT INTO students VALUES
+('Amy Smith', '123 Park Ave, San Jose', 111111);
+
+INSERT OVERWRITE students VALUES
+('Amy Smith', '123 Park Ave, San Jose', 111111);
+
+-- Multi-Row Insert Using a VALUES Clause
+INSERT INTO students VALUES
+('Bob Brown', '456 Taylor St, Cupertino', 222222),
+('Cathy Johnson', '789 Race Ave, Palo Alto', 333333);
+
+INSERT OVERWRITE students VALUES
+('Bob Brown', '456 Taylor St, Cupertino', 222222),
+('Cathy Johnson', '789 Race Ave, Palo Alto', 333333);
+
+-- Insert Using a SELECT Statement
+INSERT INTO students PARTITION (student_id = 444444)
+SELECT
+    name,
+    address
+FROM persons WHERE name = "Dora Williams";
+
+INSERT OVERWRITE students PARTITION (student_id = 444444)
+SELECT
+    name,
+    address
+FROM persons WHERE name = "Dora Williams";
+
+-- Insert Using a TABLE Statement
+INSERT INTO students TABLE visiting_students;
+
+INSERT OVERWRITE students TABLE visiting_students;
+
+-- Insert Using a FROM Statement
+INSERT INTO students
+FROM applicants
+SELECT
+--name, address, id
+applicants
+WHERE qualified = TRUE;
```

Comment:
Is this right? If selecting from `applicants` then how can you select `applicants`?

```suggestion
INSERT INTO students
FROM applicants
SELECT
name, address, id
WHERE qualified = TRUE;
```

Reply:
It's odd syntax but valid. Additional info on comment above

Reply:
What does that mean out of curiosity? Select all the columns out of that table?

Reply:
I pulled this one from the Issue which was pulled from the docs. Seemed odd so ran it, tested it and it worked.

I ran it past a few other people on my team and think it's just a very, poorly named column alias rather than a table reference.

Going to remove it from the code and test cases for now. If it turns out that it's a table reference, it won't be hard to add back in when/if someone opens an issue.

Reply:
also the columns were commented out which they shouldn't have been. fixed that as well 😁

Reply:
Well if not commented out then I'm guessing it just gives the last (`id`) column the alias `applicants`. Yeah super confusing. Glad we figured that out and removed it before merging!